### PR TITLE
[FIX] sale_stock_available_info_popup: cache miss

### PR DESCRIPTION
**Pull request by chienandalu**
cc @Tecnativa TT29615

please review @CarlosRoca13 @ernestotejeda

### CODE REVIEW

File: sale_stock_available_info_popup/models/sale_order.py [hunk 1]
```diff
@@ -16,15 +16,13 @@ class SaleOrderLine(models.Model):
     @api.depends("product_id", "product_uom_qty")
     def _compute_immediately_usable_qty_today(self):
         qty_processed_per_product = defaultdict(lambda: 0)
-        remaining = self.env["sale.order.line"]
+        self.immediately_usable_qty_today = False
```
Comment (pedrobaeza) [inline sale_stock_available_info_popup/models/sale_order.py:19]: False or 0?

Comment (chienandalu) [reply sale_stock_available_info_popup/models/sale_order.py:19]: Works either way. Odoo sets similar fields to `False` by default so I stuck with that choice

File: sale_stock_available_info_popup/models/sale_order.py [hunk 2]
```diff
@@ -16,15 +16,13 @@ class SaleOrderLine(models.Model):
...
             if not line.display_qty_widget:
-                remaining |= line
                 continue
             product = line.product_id
             qty_processed = qty_processed_per_product[product.id]
             line.immediately_usable_qty_today = (
                 product.immediately_usable_qty - qty_processed
             )
             qty_processed_per_product[product.id] += line.product_uom_qty
-        remaining.write({"immediately_usable_qty_today": False})
```
Comment (pedrobaeza) [inline sale_stock_available_info_popup/models/sale_order.py:30]: @ernestotejeda a write should never be done in a `compute`, but `update`. I think this is also happening in v12.

**Review by CarlosRoca13 — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please remove the write thing in 12.0 as well

/ocabot merge patch

**Pull request merged**